### ISSUE TITLE
Missing high contrast hover background color on dropdown menu

### ISSUE THREAD

**Issue by andresgalante** [patternfly/patternfly-next#380]
On #367 we've notices that the dropmenu item hover are not visible on hight contrast mode on edge.

![dropdownexample](https://user-images.githubusercontent.com/1832037/41549688-089868d8-72fd-11e8-9572-3a29dccc50a8.gif)

**Comment by michael-coker** [comment]
It looks like this is a potential issue with other things that use backgrounds, like buttons, badge (no visual distinction between read/unread), alert (no separation between icon area and text).

Would we want to use a media query for `-ms-high-contrast`?

One option is to underline the text on hover

```
@media screen and (-ms-high-contrast: active) {
        text-decoration: underline;
      }
```
Another could be to introduce a background for the different modes.

```
      @media screen and (-ms-high-contrast: black-on-white) {
        background: #000;
      }
      @media screen and (-ms-high-contrast: white-on-black) {
        background: #fff;
      }
```

And a side note, while looking at other things in high contrast mode, there is no box around alerts, popovers, or tooltips. Should I open a separate issue for that?

**Comment by rachael-phillips** [comment]
@mcoker Did we end up addressing this one or will we need to reopen the conversation?

**Comment by mcoker** [comment]
@rachael-phillips it's still an issue, and also an issue with other variations of dropdown components like select, options menu, app launcher, context selector, etc. This general issue would apply to other components that change background-color on hover/focus with no other sort of indicator. For example the nav component has the same issue, and I think that's a much higher priority. The accordion component applies, too, and probably others.

This might be a larger issue of reviewing the entire system in high contrast mode. As an example, the switch component toggle and down arrow/danger icons on form-control selects are invisible.

<img width="1348" alt="screen shot 2019-02-14 at 10 54 47 am" src="https://user-images.githubusercontent.com/35148959/52803221-3b775b80-3047-11e9-8a43-15e2eab21b59.png">
<img width="1348" alt="screen shot 2019-02-14 at 10 53 36 am" src="https://user-images.githubusercontent.com/35148959/52803222-3b775b80-3047-11e9-84c6-9c83a5f86108.png">

**Comment by mcarrano** [comment]
@mcoker any reason to keep this open? We have an open design issue around dark theme. Is this related or something else?

**Comment by mcoker** [comment]
WDYT @jessiehuff? This is a general issue with anything that uses a background color to indicate a state change, since background colors are ignored in high contrast mode.

**Comment by jessiehuff** [comment]
After doing some research, it doesn't seem like there's a super clear path forward from an accessibility standpoint. It seems difficult to take into account all of the OS settings and variations. I found this [thread](https://github.com/w3c/wcag/issues/623) discussing it which I thought was interesting, but unless we have a product need, I'm ok with not making this issue a high priority. I think the amount of work needed for finding all of the high contrast mode issues letting alone fixing them would be more time consuming than it may be worth.
